Add Qlik

ambster-public opened the pull request:
https://github.com/ambster-public/awesome-qlik

Qlik is a [leading](https://www.qlik.com/us/gartner-magic-quadrant-business-intelligence) business intelligence platform delivering solutions for data visualization, analytics, and reporting. Running on the same engine, its main products are QlikView (for guided analytics) and Qlik Sense (for self-service BI). This list is a curated collection of guides, blogs, extensions, and resources to make it easier for developers and analysts to make sense of data using Qlik.

### By submitting this pull request I confirm I've read and complied with the [Creating Your Own List](https://github.com/sindresorhus/awesome/blob/master/create-list.md) and [List Guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) requirements 🖖

PRs Reviewed:

* [#1605 (comment)](https://github.com/sindresorhus/awesome/pull/1605#issuecomment-525586518)
* [#1606 (comment)](https://github.com/sindresorhus/awesome/pull/1606#issuecomment-525340361)

Thanks.

ambster-public commented:
unicorn

liuchong commented:
It's a company product 😹

ambster-public commented:
> It's a company product

Company named Qlik. Still software whose resources are searched and should be better grouped.

andreykh1985 commented:
Seems deserving and complying with the [requirements](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md), would probably research for some more good extensions if there are some but these already look as really fitting awesomeness rules, no lint mistakes found.

CompSciLauren commented:
This is nice! It seems a little short compared to most lists I've looked at, but I don't know if it really matters since the idea for awesome lists is quality over quantity. It might be nice to add a description to each extension. I'm not familiar with Qlik but this looks like a useful repo.

sindresorhus commented:
@ambster-public ⬆️

ambster-public commented:
> I agree with the other comments re: format, but the amount of content does appear too thin. If I was trying to get started with Qlik, I would probably start with the official help pages first, so including them here is helpful but not doesn't connect me to a resource I wouldn't find otherwise. The blogs are nice to know about, as are the subreddits.

Yeah, I am going to work on the content some more once I've come to know some new best things. I also hope for a help from the community which will definitely come once this awesome list gets its way into the main all-embracing awesome list. For the time being, I've added another awesome Qlik blog. As far as the official help is concerned, well, I would not be so sure that it is not needed to be mentioned; I just do not think that's not really too obvious; moreover, I believe the links to these particular official sources should be present here if we want to help beginners.

> Are there other resources out there for learning about Qlik? Online courses? Books? Prominent use cases? I don't know much about the resource ecosystem out there for this platform, so it's possible that there either isn't a lot of material out there outside what you've included in the list, or perhaps much of what's out there isn't truly "awesome". Either way, with a list this short it's difficult to tell.

I've partially answered that above, and yes, I am just trying to put only truly awesome stuff on the list, hoping to get help from the community once the list becomes kind of official in terms of its awesomeness. Your suggestion about books was what I was going to do by myself, and now I've finally completed that planned work and added the "best books" list. Check it out.

> Also may be nice to extend "BI" out to "business intelligence" - I know this is common usage to some but it's not universal.

Everyone looking for resources about Qlik or any big data things knows what BI stands for as it is already too common. Not really sure this is critical indeed anyway.

> Would love to see more additions! (I didn't even know what Qlik was, beforehand!)

Thank you so much for your comments! You are welcome to check out the updated list.

ambster-public commented:
> This is nice! It seems a little short compared to most lists I've looked at, but I don't know if it really matters since the idea for awesome lists is quality over quantity. It might be nice to add a description to each extension. I'm not familiar with Qlik but this looks like a useful repo.

Thank you for the comments. My purpose is including only the best, most awesome resources whose awesomeness can hardly be put in doubt. I am going to work on that more, for sure, and I also hope for a help from the huge Qlik community once the list is added and gets more kind of publicity. Anyway, it is already updated and longer, so don't miss out on that. Thanks again!

ambster-public commented:
> Seems deserving and complying with the [requirements](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md), would probably research for some more good extensions if there are some but these already look as really fitting awesomeness rules, no lint mistakes found.

Thanks for your feedback! I am currently testing a few new ones and will consider adding more afterward. Thanks again!

ambster-public commented:
> @ambster-public

@sindresorhus Hi, thanks for your attention. Looking forward to seeing the list included. Please let me know if there's anything else I should add or change from your point of view. Thank you!

ambster-public commented:
> Glad to see the updates - the list looks much more informative now.

I agree it looks even better now. Thank you so much for your participation!

michalczaplinski commented:
While not specifically outlined in the guidelines, I would also add urls for the books listed. I don't want to single out any booksellers specifically but a clickable link to a table of contents for a book would help usability of the list.

ambster-public commented:
> While not specifically outlined in the guidelines, I would also add urls for the books listed. I don't want to single out any booksellers specifically but a clickable link to a table of contents for a book would help usability of the list.

Yeah, links to those awesome books could be helpful, so I've added them, thanks.

ambster-public commented:
@sindresorhus Does it look good enough now or what else should I do about it so it can get added to your awesome list of awesome lists? Please advise. Thanks.

sindresorhus commented:
```
❯ awesome-lint https://github.com/ambster-public/awesome-qlik
✖ Linting

  README.md:36:154
  ✖  36:154  List item link and description must be separated with a dash  remark-lint:awesome-list-item
  ✖  37:116  List item link and description must be separated with a dash  remark-lint:awesome-list-item
  ✖  38:125  List item link and description must be separated with a dash  remark-lint:awesome-list-item
  ✖  39:125  List item link and description must be separated with a dash  remark-lint:awesome-list-item
  ✖  40:131  List item link and description must be separated with a dash  remark-lint:awesome-list-item
  ✖  41:110  List item link and description must be separated with a dash  remark-lint:awesome-list-item
  ✖  42:123  List item link and description must be separated with a dash  remark-lint:awesome-list-item
  ✖  46:115  List item link and description must be separated with a dash  remark-lint:awesome-list-item
  ✖  48:2    List item link and description must be separated with a dash  remark-lint:awesome-list-item
  ✖  49:117  List item link and description must be separated with a dash  remark-lint:awesome-list-item
  ✖  50:111  List item link and description must be separated with a dash  remark-lint:awesome-list-item
  ✖  51:100  List item link and description must be separated with a dash  remark-lint:awesome-list-item
  ✖  52:123  List item link and description must be separated with a dash  remark-lint:awesome-list-item
  ✖  53:113  List item link and description must be separated with a dash  remark-lint:awesome-list-item
  ✖  54:135  List item link and description must be separated with a dash  remark-lint:awesome-list-item

  15 errors
```

sindresorhus commented on readme.md:
```diff
@@ -313,6 +313,7 @@
 - [Data Engineering](https://github.com/igorbarinov/awesome-data-engineering#readme)
 - [Streaming](https://github.com/manuzhang/awesome-streaming#readme)
 - [Apache Spark](https://github.com/awesome-spark/awesome-spark#readme) - Unified engine for large-scale data processing.
+- [Qlik](https://github.com/ambster-public/awesome-qlik/) - BI platform for data visualization, analytics, and reporting apps.
```
```suggestion
- [Qlik](https://github.com/ambster-public/awesome-qlik#readme) - Business intelligence platform for data visualization, analytics, and reporting apps.
```

ambster-public commented:
@sindresorhus Missed that, sorry. Now it's passed.
`✔ Linting`

sindresorhus commented:
Tweet: https://twitter.com/awesome__re/status/1195615855863726081
